expandybird/Makefile

```diff
@@ -47,3 +49,21 @@ clean:
 	-docker rmi $(PREFIX)/$(IMAGE):$(TAG)
 	rm -f expandybird
 
+.PHONY: test
+test: lint vet test-unit
+
+.PHONY: test-unit
+test-unit:
+	$(GO_TEST)
+
+.PHONY: lint
+lint:
+	$(GO_LINT)
+
+.PHONY: vet
+vet:
+	$(GO_VET)
```

Comment:
These commands actually could all live in include.mk for less duplication. :smile: